internal/prometheus/prometheus.go

```diff
@@ -149,6 +149,21 @@ func (p *Provider) Clear() {
 	p.inner.Clear()
 }
 
+// Register register the collectors on OPA prometheus registry
+func (p *Provider) Register(c prometheus.Collector) error {
+	return p.registry.Register(c)
+}
+
+// MustRegister register the collectors on OPA prometheus registry and panics when an error occurs
+func (p *Provider) MustRegister(cs ...prometheus.Collector) {
+	p.registry.MustRegister(cs...)
+}
+
+// Unregister unregister the collectors on OPA prometheus registry
```

Comment:
Similarly here, do you think we'll need the ability to unregister a collector?

Reply:
the same explicable above .